packages/bff/src/services/catalogService.ts

```diff
@@ -708,9 +712,9 @@ export function catalogServiceBuilder(
         doc: catalogApi.EServiceDoc
       ): Promise<catalogApi.CreateEServiceDescriptorDocumentSeed> => {
         const clonedPath = await fileManager.copy(
-          config.s3Bucket,
-          config.eserviceDocumentsPath,
+          config.eserviceDocumentsContainer,
           doc.path,
+          config.eserviceDocumentsPath,
```

Comment:
This was a bug, the target base path is the third argument, not the second one

Reply:
we can follow the same approach of definition [`FileManager.storeBytes`](https://github.com/pagopa/interop-be-monorepo/blob/f30e92a54d47f4f016d6104edcdffc0fed7e5868/packages/commons/src/file-manager/fileManager.ts#L38) using an structured object as param instead? That way we can avoid these errors types

Reply:
Yes we could! But I didn't want this PR to touch all the services with that change :) Maybe we could track this as an improvement